_layouts/docs.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+---
+
+<section class='jump-to-release'>
+  <div class='container '>
+    <div class='row'>
+      <div class='twelve columns'>
+      <h1>Electron Documentation</h1>
+      {% if page.category != 'ignore' %}
+      <a href='/docs/{{ page.version }}/'>{{ page.version }} {% if site.latest_version == page.version %} (Latest) {% endif %} </a> |
+      <a href='{{ page.source_url }}'>Document Source</a>
+      {% endif %}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class='docs'>
+  <div class='container'>
+    <div class='row'>
+      <div class='ten columns release offset-by-one'>
+        {% if page.category != 'ignore' %}
+        <h6 class='docs-breadcrumb'>{{ page.version }} / {{ page.category }}
+          {% if page.title != 'README' %} / {{ page.title }} {% endif %}</h6>
+        {% endif %}
+
+        {{ content }}
+
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='ten columns release offset-by-one docs-footer'>
+        <span class="octicon octicon-pencil"></span>
+        See something that needs fixing? Propose a change on the <a href='{{ page.source_url }}'>source</a>.
```

Comment:
Oh yeah. Since the Available Versions page isn't one that correlates to a page in the `atom/electron` and the second link links to that page, I'm going to remove that section for this page.